apiserver/facades/agent/caasapplication/application_test.go

```diff
@@ -90,6 +90,7 @@ func (s *CAASApplicationSuite) SetUpTest(c *gc.C) {
 		s.ControllerDomainServices(c).ControllerConfig(),
 		s.applicationService,
 		s.ControllerDomainServices(c).Config(),
+		&mockModelAgent{},
```

Comment:
I would lean towards using gomocks for new code, I don't know if it matters too much here though.

Reply:
Agreed, but it was more difficult to add in given how someone started using the other approach.